scripts/normalize-by-median.py

```diff
@@ -72,25 +76,25 @@ def WithDiagnostics(ifile, batch_size, fp, paired, norm):
 
             print('... in file ' + input_filename, file=sys.stderr)
 
-            if report_fp:
+            if fp:
                 print(total + " " + total - discarded + " " +
                       1. - (discarded / float(total)), file=fp)
                 report_fp.flush()
 
         # If in paired mode, check that the reads are properly interleaved
-        if paired:
-            if not check_is_pair(batch[0], batch[1]):
-                raise IOError('Error: Improperly interleaved pairs \
-                    {b0} {b1}'.format(b0=batch[0].name, b1=batch[1].name))
-
+        if paired and not paired_reads:
```

Comment:
This is confusing language... suggest 'force_paired' for first name, 'is_paired' for second variable name.